test/run

```diff
@@ -35,8 +35,14 @@ case "${TEST_SCENARIO:=}" in
     *expensive*)
         RUN_OPTS="$(echo "$ALL_TESTS" | grep -E "$RE_EXPENSIVE")"
         ;;
+    *cockpit*)
```

Comment:
This way of splitting the tests of course has the risk of putting some tests into more than one scenario. For example, there might very well be a test names TestStorageE2E which would end up both in the "expensive" and "storage" scenario. We have the same issue in Cockpit's own test/run and it would probably be good to find a better approach.

But I guess this works for now.

Reply:
OH true, good catch. we do have such tests. I will adjust to exclude the expensive tests in the storage and cockpit scenarios.